README.md

```diff
@@ -526,6 +526,52 @@ Sunspot.search(Post) do
 end
 ```
 
+#### Json Facets
+
+The [json facet](http://yonik.com/json-facet-api/) can be used with the following syntaxt:
+
+```ruby
+Sunspot.search(Post) do
+  json_facet(:title)
+end
+```
+
+There are some options you can pass to the json facet:
```

Comment:
lets highlight these as a code: `limit` etc

Reply:
@baldarn I think this one it the last one ☝️

Reply:
I didn't get this. do you want me to do some examples?

Reply:
@baldarn yeah, examples for each option would be nice